src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -4545,6 +4545,42 @@ class C
             End Using
         End Function
 
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestSuggestionMode(completionImplementation As CompletionImplementation) As Task
```

Comment:
This really wasn't tested in Old Completion? Or is there some Old Completion-only test somewhere that wasn't updated to be used in both suites?

Reply:
Or was it tested in integration tests only (and Sam found it that way or something)?

Reply:
This really was not tested in the old completion.